pyproject.toml

```diff
@@ -11,7 +11,7 @@ keywords    = ["automation", "tasks", "task-runner", "plugin", "poetry", "poe"]
 classifiers = ["Topic :: Software Development :: Build Tools"]
 
 [tool.poetry.dependencies]
-python = ">=3.7"
+python = ">=3.7,<4.0"
```

Comment:
I had to make this change because Poetry would complain about the Sphinx version being incompatible with `python >=4.0`. As long as `poetry install --with=dev` works and installs Sphinx, it's all good 👍🏻